### PR TITLE
Add PowerShell scripts used for DC/OS Windows CI/CD

### PR DESCRIPTION
**Pull request by ionutbalutoiu**
This pull request adds the following PowerShell scripts to be used among the testing jobs for DC/OS Windows CI / CD:

* `windows-ci-server-setup.ps1` - Performs an unattended installation of all the prerequisites needed for the DC/OS Windows build and Tox unit tests. It should be run only once when setting the up the Windows CI agent.
* `dcos-windows-buid.ps1` - Used to perform a local DC/OS Windows build. It assumes that the current working directory is a checkout of the upstream [dcos/dcos](https://github.com/dcos/dcos) repository. All the prerequisites must be already installed before executing this script.
* `dcos-windows-tox-tests.ps1` - Used to execute the DC/OS Windows Tox unit tests. It assumes that the current working directory is a checkout of the upstream [dcos/dcos](https://github.com/dcos/dcos) repository.
* `run-unit-tests.ps1` - Generalized PowerShell script to run the Windows unit tests against any of the 4 projects: `dcos-go`, `dcos-metrics`, `dcos-diagnostics` and `dcos-net`.
  
  You need to locally check-out the Git repository and execute the script as such:
  ```
  .\run-unit-tests.ps1 -Component "dcos-net" -Directory "<local_git_repository>"
  ```
  
  where `<local_git_repository>` points to the local Git directory of the component specified via `Component` parameter.

### CODE REVIEW

File: scripts/dcos-windows-tox-tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,43 @@
+$ErrorActionPreference = "Stop"
+
+
+function Confirm-DCOSToxRequirements {
+    # - Make sure the correct Python version is installed
+    $pythonVersion = python.exe --version
+    if($LASTEXITCODE) {
+        Throw "Failed to get the Python version"
+    }
+    if($pythonVersion -ne "Python 3.6.5") {
```
Comment (pcrews) [inline scripts/dcos-windows-tox-tests.ps1]: nitpick: Would like to see the expected version in a variable and the error output to use the variable to make it easier to change things, but this is a very minor nice-to-have.

File: scripts/dcos-windows-tox-tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        Throw "Python has wrong version! Expected at least 3.6.5 and found: $pythonVersion"
```
Comment (pcrews) [inline scripts/dcos-windows-tox-tests.ps1]: The phrasing of the error message could be confusing.
The test looks for a hard match of "Python 3.6.5", but the message implies that newer versions would succeed, though the logic implies they would not.
Something like "Expected: $expectedPythonVersion but found: $pythonVersion"

**Review by pcrews — APPROVED** [review]
Minor quibbles on a single, small chunk of code, but otherwise LGTM!

**Comment by ionutbalutoiu** [comment]
@pcrews Thank-you for the review! I fixed all your comments.

**Comment by pcrews** [comment]
> @pcrews Thank-you for the review! I fixed all your comments.

You are most welcome, the changes look awesome, and thank you very much :smiley:

**Review by orsenthil — APPROVED** [review]
LGTM.

**Pull request merged**